include/uv-win.h

```diff
@@ -78,6 +78,7 @@ typedef intptr_t ssize_t;
 # define SIGABRT_COMPAT       6
 #endif
 
+UV_EXTERN int uv_translate_win32_error(int sys_errno);
```

Comment:
this is not the place for `UV_EXTERN`

both function should be called the same for example: `uv_translate_os_error`
`UV_EXTERN uv_translate_os_error` only on `uv.h`